### PR TITLE
Triangle normal

### PR DESCRIPTION
**Pull request by jackbeagley**
Implements #153

### CODE REVIEW

File: src/polytopes/ngon.jl [hunk 1]
```diff
@@ -122,6 +122,21 @@ function Base.in(p::Point{Dim,T}, ngon::Ngon{N,Dim,T}) where {N,Dim,T}
...
+Determine the normalised normal of `triangle` in three
+dimensions
+"""
+function normal(t::Triangle{3})
+  a, b, c = t.vertices
+
+  # the normal of a Triangle is the cross product of two arbitrary edges
+  t_normal = (b - a) × (c - a)
+
+  t_normal / norm(t_normal)
```
Comment (juliohm) [inline src/polytopes/ngon.jl]:
```suggestion
  a, b, c = t.vertices
  n = (b - a) × (c - a)
  n / norm(n)
```

File: src/Meshes.jl [hunk 2]
```diff
@@ -155,7 +155,7 @@ export
   windingnumber, chains, segments,
   isclosed, issimple, hasholes,
   angles, innerangles, close!, open!,
-  orientation, bridge,
+  orientation, bridge, normal
```
Comment (juliohm) [inline src/Meshes.jl]:
```suggestion
  orientation, bridge, normal,
```

**Comment by jackbeagley** [comment]
Thanks for catching that @juliohm.. I couldn't at all work out why the tests weren't passing!

**Comment by juliohm** [comment]
Awesome contributions! 🎉

**Pull request merged**